src/cmdlib.py

```diff
@@ -19,3 +20,10 @@ def sha256sum_file(filename):
         for b in iter(lambda: f.read(128 * 1024), b''):
             h.update(b)
     return h.hexdigest()
+
+def fatal(msg):
```

Comment:
Is this purely for future use and nothing was wanting to print an error & `sys.exit(1)` yet?

Reply:
Yeah, I figured it would get use in the future; actually surprised nothing needs this yet.

I modeled it after https://github.com/coreos/coreos-assembler/blob/master/src/virt-install#L12

I'd just as soon use `sys.exit(1)` instead of `SystemError`

Reply:
Looks like they're the same thing. `sys.exit(1)` just raises `SystemExit` too, so they're mostly equivalent: https://docs.python.org/3/library/sys.html#sys.exit.